hladnymatfyzak/__init__.py

```diff
@@ -39,6 +39,21 @@ def __repr__(self):
                         ")>")
 
 
+def _meals_from_bs(data, type, opts):
```

Comment:
@mrshu : Could you just rename variable data? To something that hints it is string type, that is later used in regex. For example html_data or site_source...